src/modal/modalContent/signIn.js

```diff
@@ -0,0 +1,26 @@
+import { useDispatch } from "react-redux";
+import { closeModal } from "../../store/showmodal";
+import "./index.css";
+function SigniInUser() {
+  const dispatch = useDispatch();
+  return (
+    <>
+      <div className="signiindiv">
+        <input className="signin" placeholder="name..." />
+        <input className="signin" placeholder="email..." />
+        <input className="signin" placeholder="password..." />
+        <input className="signin" placeholder="confirm password..." />
+      </div>
+
+      <button className="btn btn-outline-dark"> Create account </button>
```

Comment:
On click should add user with ability to sign in.

Reply:
ok